### PR TITLE
Fixed KeyError exception on tags without href attr

### PR DESCRIPTION
**Pull request by defsen**

### CODE REVIEW

File: bleach/callbacks.py [hunk 1]
```diff
@@ -4,7 +4,11 @@
 
 def nofollow(attrs, new=False):
     href_key = (None, u'href')
-    if href_key not in attrs or attrs[href_key].startswith(u'mailto:'):
+
+    if href_key not in attrs:
+        return attrs
+
+    if attrs[href_key].startswith(u'mailto:'):
```
Comment (willkg) [inline bleach/callbacks.py]: Why'd you make this change? It doesn't seem like it changes the behavior at all.

File: bleach/callbacks.py [hunk 2]
```diff
@@ -18,6 +22,10 @@ def nofollow(attrs, new=False):
 
 def target_blank(attrs, new=False):
     href_key = (None, u'href')
+
+    if href_key not in attrs:
+        return attrs
```
Comment (willkg) [inline bleach/callbacks.py]: Good catch!

Comment (willkg) [inline bleach/callbacks.py]: Looks like `target_blank` has no tests. Can you add some?

File: bleach/callbacks.py (hunk 1 above)
Comment (defsen) [inline bleach/callbacks.py]: Oops, missed it.

File: bleach/callbacks.py (hunk 2 above)
Comment (defsen) [inline bleach/callbacks.py]: Of course.

Comment (willkg) [inline bleach/callbacks.py]: Mmm... It looks like neither of these callbacks have any tests. That's terrible. Sorry about that. I'll throw some together after landing this.

**Comment by willkg** [comment]
Thank you for fixing this!

**Pull request merged**